### PR TITLE
Update release numbers for 5.2.0

### PR DESCRIPTION
**Pull request by tst-lsavoie**
This PR updates the version numbers for 5.2.0.

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
I've now tested this PR.

File: earth_enterprise/src/fusion_version.txt [hunk 1]
```diff
@@ -20,5 +20,5 @@
 # Format is 2 lines (comment lines ignored):
 # 1st) full version number: e.g., 3.1.2.3,
 # 2nd) is short number (i.e., drop trailing 0's, i.e., 3.1.0 -> 3.1, or 4.0.0 -> 4)
-5.1.3
-5.1.3
+5.2.0
+5.2.0
```
Comment (tst-awankhede) [inline earth_enterprise/src/fusion_version.txt]: based on comments , should this be short form 5.2 ? either that or comments need a change.

Comment (tst-lsavoie) [inline earth_enterprise/src/fusion_version.txt]: Looking at how it's used in scons the second line isn't needed.  I'm testing that right now but it's going to take a while to run a full build.

**Comment by tst-ccamp** [comment]
When I tested I get this:
```
This machine is running fusion software version 5.2.0.
/gevol/assets is configured to use version 5.1.3.
In order to use /gevol/assets from this machine, it must be upgraded.
Run the following command:
  geupgradeassetroot --assetroot /gevol/assets
```

Do we know what this command will do? Is it actually needed or is the check being run incorrectly? @tst-awankhede @tst-lsavoie

**Comment by tst-awankhede** [comment]
what does your /gevol/assets/.config/.fusionversion have ?

**Comment by tst-ccamp** [comment]
It says 5.1.3, as expected. So, it always has to update the number even if there aren't any actual changes to the asset root format?

**Comment by tst-ccamp** [comment]
I've tested and it looks like the version number has been updated everywhere I expect it to be. I just have that question about geupgradeassetroot. But, after that, I think it's good to go.

**Comment by tst-ccamp** [comment]
@tst-awankhede Since we're waiting on this one in order to get started on the testing, I'm going to go ahead and merge it. @tst-lsavoie can make any other changes required in a separate PR.

**Comment by tst-lsavoie** [comment]
@tst-ccamp are we good on your question about geupgradeassetroot?

**Pull request merged**